helpers/notifications.js

```diff
@@ -135,9 +136,14 @@ const scheduleNotifications = async (req, res) => {
         console.error(error);
       }
     }
+<<<<<<< HEAD
+    return res.send("success");
+  })();
+=======
   })();
 
   return res.send("success");
+>>>>>>> 30d77096642d2767332e51f5538f88ea3c4c940f
```

Comment:
Let's remove this merge conflict markers

Reply:
I think my new message that caused this conflict was something like: "Success! Notifications have been sent."